pkg/detectors/tailscaleapi/tailscaleapi.go

```diff
@@ -0,0 +1,85 @@
+package tailscaleapi
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(`^tskey-(api|oauth)-[0-9A-Za-z_]+-[0-9A-Za-z_]+$`)
```

Comment:
The comment above makes me believe that they want to use word boundaries and not start/end anchors.

```suggestion
	keyPat = regexp.MustCompile(`\btskey-(api|oauth)-[0-9A-Za-z_]+-[0-9A-Za-z_]+\b`)
```